config/custom-environment-variables.yaml

```diff
@@ -340,6 +340,7 @@ notifications:
 
 coverage:
   plugin: COVERAGE_PLUGIN
+  enabled: COVERAGE_PLUGIN_ENABLED
```

Comment:
I think `COVERAGE_PLUGIN_ENABLED` looks "we can use coverage plugin or not". 
Do we use `default: COVERAGE_PLUGIN_DEFAULT_ENABLED` or another one?